lms/djangoapps/courseware/views.py

```diff
@@ -131,7 +130,10 @@ def courses(request):
         else:
             courses_list = sort_by_announcement(courses_list)
 
-        return render_to_response("courseware/courses.html", {'courses': courses_list})
+    return render_to_response(
```

Comment:
Nit: we generally use ``json.dumps`` at the point in the template where it is needed. It isn't clear that the ``course_discovery_meanings`` parameter is an encoded string.